sw/airborne/modules/stereocam/stereocam.c

```diff
@@ -76,6 +111,9 @@ extern void stereocam_disparity_to_meters(uint8_t *disparity, float *distancesMe
 
 extern void stereocam_start(void)
 {
+  struct FloatEulers euler = {RadOfDeg(STEREO_BODY_TO_STEREO_PHI), RadOfDeg(STEREO_BODY_TO_STEREO_THETA), RadOfDeg(STEREO_BODY_TO_STEREO_PSI)};
```

Comment:
Yes, it will read the value from the xml file as deg, but make the define in rad, see https://wiki.paparazziuav.org/wiki/Units